### ISSUE TITLE
it takes about 40 seconds to see done

### ISSUE THREAD

**Issue by Archer-Thane** [webrtc/samples#1205]
dear all
Thanks for your time and energy on this.
Anyone out there using your trickle ICE to check STUN/TURN servers.
I do it too. but today, something unusual is taking place I wondered to share with you. 
Everything was fine until today. I opened up the web page and trying to check my turn server. 
It takes 40 seconds to show me done ( which was about 1 or 2 seconds previously).
I thought it is my server's bug. so I changed the address to google's stun server.
And 40 seconds remains to be persistent. 
That seems really weird to me, because I have tested it on my phone and everything is fine (result is prepared between 1 to 2 seconds)
I tested it with Opera on my laptop, and result takes 40 seconds to be prepared.
I saw new commits, so I thought I should report this.
Thanks again and hope this be useful

**Comment by Archer-Thane** [comment]
also an image of what happens: 
https://ibb.co/PMcFNTW

**Comment by danjenkins** [comment]
Hey @Archer-Thane .

The 40 seconds is a default timeout. I expect your computer for whatever reason is having some sort of network connectivity issue with stun/turn. Have you tried it whilst being on a different network etc? Was your phone on the same network?

**Comment by Archer-Thane** [comment]
Hey @danjenkins .
Thanks for reply.
No my phone is on another network. yeah you're right. changing phone to the same net that laptop is on, makes phone to behave like laptop.
So the problem is network related.
Thanks.

**Comment by danjenkins** [comment]
Glad I could help - I don't have permissions on this repo to close the issue - can you close it?

**Comment by hkn-tr** [comment]
**Virtual Ethernet Adapters are causing a 40 second delay.**

After creating the offer and answer on stun server, I checked the sdp key data of the description, I noticed that there were more than one ip addresses and I checked them one by one from cmd>ipconfig. Some of them belonged to virtual ethernet adapters.

Win+X > Device Manager > Network Adapters

On Device Manager, I right clicked on the Virtual Ethernets and disabled them one by one,then work like a charm, the create offer and create answer operations are performed within 2 seconds at every time.

Now the next problem is: how can we solve this problem without expecting the users to disable virtual ethernet adapters.
